### DISCUSSION TITLE
Will RefindPlus Provide Nightly Builds?

### DISCUSSION THREAD

**Discussion by atomizasser** [dakanji/RefindPlus#39] [General]
Hi. Ive discover this project and i very interested. I have a pc with dual boot, windows and macos. I’ve use opencore to boot macos but i have some troubles to boot windows using opencore. Ive tested last refindplus release but seems that ui is not scaled correctly and icons appears at big size (I’ve  active ui scale and reduce icon size to 128 px and 42 px).
I’ve seen that you has committed  a lot of changes since last release oriented to opencore combination, would be possible test some nightly build to test it?
Thanks! Regards

**Comment by dakanji** [comment]
> I’ve seen that you has committed a lot of changes since last release oriented to opencore combination

Just to clarify that the commits made are not orientated to OpenCore. 

Anything added is for the improvement of the application whether run alone or in combination with OpenCore or anything else.

> test some nightly build

No plans to implement this at this time as it is not expected that changes will continue to be as frequent over time. 

Note that it is possible to implement this yourself by scheduling a script that will update the two repos and run the build script on your computer.

**Reply by atomizasser** [reply]
Thanks for your answer. I will wait until next release. I have seen that you usually release a new vrsion at the beginning of the month. Is that right?
Another question: do you know why rewsize icons to 128px is not working? Ive edited de .conf to force it but still show icons at default 256px. Thanks

**Comment by dakanji** [comment]
> I have seen that you usually release a new version at the beginning of the month. Is that right?

Yes, currently around the beginning of each month. It might change to every other month when things start to slow down.

> do you know why resize icons to 128px is not working? 

Sorry, I don't know exactly at the moment. Some users have had display issues due to their monitor setting (FreeSync on) in the past. You can try with another monitor to see if there is a difference. You can also try the standard rEFInd to see if it behaves differently. If it does, you can raise an issue and provide the information requested in the issue template.
